client.py

```diff
@@ -41,25 +42,28 @@ async def on_message(message):
         read_games_mapping()
     elif user_input == 'games_m':
         log_wishlist_memory()
+    # TODO make 'delete' and 'add' fucntions take regex to grab everything in the quotes.
+    # This will throw all logic into functions to only have a function call here.
     elif 'delete' in user_input:
         game_in_input = re.split('"',user_input)
         game_name = game_in_input[1]
         entry = get_entry(game_name.lower())
         if game_name != None:
             await handle_delete_game_request(message,entry)
-    elif 'clear' in user_input: 
+    elif 'add' in user_input:
+        game_in_input = re.split('"',user_input)
+        game_name = game_in_input[1]
+        entry = get_entry(game_name.lower())
+        if entry != None:

```

Comment:
This check is redone in the handle_add_game_request function

Reply:
So if this fails we'll just not receive a reply, instead of the desired "check gamepass" message